websites/K/Komik Indo/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "242969117479403520",
+		"name": "1t9"
+	},
+	"service": "Komik Indo",
+	"description": {
+		"en": "KomikIndo is an online comic reading site with the most complete and updated collection. You can read thousands of comic collections that we update every day for free. With a responsive and modern design, this website is the best place for those of you who want to read the latest Manga, Manhwa (Korean comics) and Manhua (Chinese comics)."
+	},
+	"url": "komikindo.tv",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/HccQ60t.png",
```

Comment:
The [logo](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) should be a 512x512px image.